api/client/samples/crop_w_series/soybean_w_drought_index.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>The comment in this cell can be a separate markdown cell:</p><ol><li>Drought Index</li></ol><ul><li>Three options are available: daily, weekly or monthly</li><li>We can also see the information related to each available frequency below</li></ul><p>and then similarly for Crop Acreage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done</p>